src/netius/test/servers/socks.py

```
@@ -186,3 +187,333 @@ def _make_connection(self):
         self.loop.connections_m[_socket] = connection
         connection.open()
         return connection
+
+
+class SOCKSServerTest(unittest.TestCase):
+
+    def setUp(self):
+        unittest.TestCase.setUp(self)
+        if mock == None:
```

Ruff (0.16.3)

[error] Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)